.github/workflows/codeql-analysis.yml

```
@@ -13,24 +13,28 @@ name: 'CodeQL'
     # The branches below must be a subset of the branches above
     branches: [main]
   schedule:
-    - cron: '27 0 * * 0'
+    - cron: '24 0 * * 1 8'
 
 jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
     permissions:
       actions: read
```

actionlint (1.7.12)

[error] "read write" is invalid as permission of scope "actions". available values are "read", "write", "none"

(permissions)